ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+    ...         Tier1
+    ...         ODS-2209
+    Create A Data Science Projects    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${TEST_USER_2.AUTH_TYPE}    ${PROJECT_USER2}    # robocop: disable:line-too-long
+    Create A Data Science Projects    ${TEST_USER_3.USERNAME}    ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}    ${PROJECT_USER3}    # robocop: disable:line-too-long
+    ${status}    Login And Wait Dsp Route    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${PROJECT_USER3}    ds-pipeline-pipelines-definition    ${1}    # robocop: disable:line-too-long
+    Should Be True    ${status} == 403    The user must not have permission to access
+    Run    oc policy add-role-to-user ds-pipeline-user-access-pipelines-definition ${TEST_USER_2.USERNAME} -n ${PROJECT_USER3} --role-namespace=${PROJECT_USER3}    # robocop: disable:line-too-long
+    ${status}    Login And Wait Dsp Route    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${PROJECT_USER3}    ds-pipeline-pipelines-definition    ${1}    # robocop: disable:line-too-long
+    Should Be True    ${status} == 200    Rolling Binding Not Working    # robocop: disable:line-too-long
+
+
+*** Keywords ***
+General Suite Setup
+    [Documentation]    Suite setup steps for testing DSG. It creates some test variables
+    ...                and runs RHOSi setup
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+
+General Suite Teardown
+    [Documentation]    General Suite Teardown
+    Remove Pipeline Project    ${PROJECT_USER2}
+    Remove Pipeline Project    ${PROJECT_USER3}
+    RHOSi Teardown
+
+Create A Data Science Projects
+    [Documentation]    Create A Data Science Projects
+    [Arguments]     ${user}    ${password}    ${auth_type}    ${project}
+    Launch Data Science Project Main Page    username=${user}
+    ...    password=${password}
+    ...    ocp_user_auth_type=${auth_type}
+    ...    browser_alias=${user}-session
+    Open Data Science Projects Home Page
+    Remove Pipeline Project    ${project}
+    Create Data Science Project    title=${project}    description=
+    Create Workbench    workbench_title=${project}-workbench
+    ...                 workbench_description=
+    ...                 prj_title=    image_name=${NB_IMAGE}   deployment_size=Small
+    ...                 storage=Persistent  pv_existent=${FALSE}    pv_name=${PV_NAME_TOL_1}
+    ...                 pv_description=${PV_DESCRIPTION}  pv_size=${PV_SIZE}
+    Create S3 Data Connection    project_title=${project}    dc_name=${project}-dc
+    ...                          aws_access_key=${S3.AWS_ACCESS_KEY_ID}
+    ...                          aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...                          aws_s3_endpoint=${S3.AWS_DEFAULT_ENDPOINT}    aws_region=${S3.AWS_DEFAULT_REGION}
+    ...                          aws_bucket_name=${S3_BUCKET}
+    Wait For Workbench Pod    ${project}    ${project}-workbench-0    ${2}    ${60}
```

Comment:
what the workbenche is used for?

Reply:
can you remove the workbench creation if not needed?

Reply:
Done